__tests__/app.test.js

```diff
@@ -130,6 +130,45 @@ describe("GET /api/reviews/:review_id", () => {
   });
 });
 
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("201, responds with the posted comment", () => {
+    const commentToAdd = { username: "mallionaire", body: "Great game!" };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(commentToAdd)
+      .expect(201)
+      .then((res) => {
+        const { comment } = res.body;
+        expect(comment).toHaveProperty("comment_id");
+        expect(comment).toHaveProperty("body", "Great game!");
+        expect(comment).toHaveProperty("votes", 0);
+        expect(comment).toHaveProperty("author", "mallionaire");
+        expect(comment).toHaveProperty("review_id", 4);
+        expect(comment).toHaveProperty("created_at");
+      });
+  });
```

Comment:
We need one more happy path test here. It is possible for our client to include an unnecessary property on the post body ie: { username: "mallionaire", body: "Great game!", votes: 50 }
In this case we should get a 201 with the unnecessary properties ignored